src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  background-color: rgb(255, 255, 255);
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
+  font-family: "Roboto", sans-serif;
+}
+
+.logo {
+  display: inline-block;
+  margin-left: 50px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  width: 40px;
+  height: 40px;
+  background-image: url(images/logo.png);
+}
+
+.navBar {
+  display: inline-block;
+  height: 60px;
+  margin-right: 50px;
+}
+
+.buttons {
+  display: flex;
+  flex-direction: row;
+  text-transform: uppercase;
+  font-size: 12px;
+  align-items: baseline;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.eachButton {
+  text-decoration: none;
+  color: black;
+  margin-left: 20px;
+  height: 60px;
+  padding-top: 22.5px;
```

Comment:
And how to change it if the height will be 80 or 90?
It is better to center the content vertically but not to fix paddings